hebiros_gazebo_plugin/include/hebiros_gazebo_pid.cpp

```diff
@@ -0,0 +1,33 @@
+#include "hebiros_gazebo_pid.h"
+
+namespace hebiros {
+namespace sim {
+
+// Update and return the new output command
+double PidController::update(double target, double feedback, double dt, const PidGainsMsg& pid_gains, size_t gain_idx)
+{
+  // "Disable" the controller if commands are nan
+  if (std::isnan(target)) {
+    return 0;
+  }
+  double error_p, error_i, error_d;
+  error_p = target - feedback;
+  error_i = _elapsed_error + error_p;
+  error_d = (error_p - _prev_error) / dt;
```

Comment:
If `dt` is +/- 0, this will be infinite. How are you wanting to handle that?

Reply:
I think this was correct as is, but I cleaned up the code to clarify what's going on here.